config/initializers/app_key.rb

```diff
@@ -0,0 +1,2 @@
+ENV['AWS_ACCESS_KEY_ID'] = "AKIAJKYEVYCXN76Y34SQ"
+ENV['AWS_SECRET_ACCESS_KEY'] = "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"
```

Comment:
BIG PROBLEM: This should be in `.gitignore`. Please revoke those keys immediately